hs_collection_resource/templates/pages/collectionresource.html

```diff
@@ -55,7 +55,15 @@ <h3>Collection Contents </h3>
                                             {% endif %}
                                         </td>
                                         <td>
-                                            {{ res.resource_type }}
+                                            {% if res.resource_type == "CompositeResource" %}
+                                                Resource
```

Comment:
unsure if we want to change "composite resource" to "resource" here:
https://github.com/hydroshare/hydroshare/blob/4853-resource-type-name-fix/hs_composite_resource/views.py#L45

Reply:
I was thinking of changing it and thought it would require probably changing at many places where we are raising error related to composite resource. I guess since these error messages are for the user, I should change it.

Reply:
all good either way for me, just checking...

Reply:
I have updated those messages where appropriate.

Reply:
Nice!